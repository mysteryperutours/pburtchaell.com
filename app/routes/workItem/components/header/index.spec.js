import React from 'react';
import WorkItemHeader from './index';

describe(WorkItemHeader.name, () => {
  it('renders component when pending', () => {
    const component = (
      <WorkItemHeader
        isPending={true}
      />
    );

    expect(shallow(component)).toMatchSnapshot();
  });

  it('renders component when fulfilled', () => {
    const component = (
      <WorkItemHeader
        isPending={false}
        title="Foo"
        description="Foo is foo"
        backgroundColor="rgb(0,0,0)"
        textColor="rgb(0,0,0)"
        coverImage={{
          url: ''
        }}
      />
    );

    expect(shallow(component)).toMatchSnapshot();
  });
});
