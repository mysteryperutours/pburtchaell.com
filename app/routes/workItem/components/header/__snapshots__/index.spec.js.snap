exports[`WorkItemHeader renders component when fulfilled 1`] = `
<header
  className="case-study-header"
  style={
    Object {
      "background": "rgb(0,0,0)",
      "backgroundAttachment": "fixed",
    }
  }>
  <Row
    defaultColumn={true}
    elementType="div"
    size="full">
    <div
      className="case-study-titles">
      <Text
        style={
          Object {
            "color": "rgb(0,0,0)",
          }
        }
        type="h1">
        Foo
      </Text>
      <Text
        style={
          Object {
            "color": "rgb(0,0,0)",
          }
        }
        type="h2">
        Foo is foo
      </Text>
    </div>
  </Row>
  <Row
    defaultColumn={true}
    elementType="div"
    size="full">
    <div
      className="case-study-cover-image"
      style={
        Object {
          "backgroundImage": "url()",
          "backgroundPosition": "top center",
          "backgroundSize": "cover",
          "height": "42rem",
          "width": "100%",
        }
      } />
  </Row>
</header>
`;

exports[`WorkItemHeader renders component when pending 1`] = `
<header
  className="case-study-header is-pending"
  style={Object {}}>
  <Row
    defaultColumn={true}
    elementType="div"
    size="full">
    <div
      className="case-study-cover-image"
      style={
        Object {
          "height": "42rem",
          "width": "100%",
        }
      } />
  </Row>
</header>
`;
